ciao-launcher/instance.go

```diff
@@ -57,12 +57,12 @@ type insStartCmd struct {
 	cfg      *vmConfig
 	rcvStamp time.Time
 }
-type insRestartCmd struct{}
 type insDeleteCmd struct {
-	suicide bool
-	running ovsRunningState
+	suicide         bool
```

Comment:
@markdryan Do you mind documenting the fields here. As they convey quite a bit of information in the code.